src/AppBundle/RSS/Parser.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+    namespace AppBundle\RSS;
+
+    /**
+      * Parser class
+      * Provides interface to parse RSS with different templates
+      * @package RSS
+    */
+    class Parser {
+
+        /**
+         * @var Template $template
+         */
+        public $template;
+        /**
+         * @var array $items
+        */
+        public $items;
```

Comment:
Items should be returned from the ``feed`` method rather than be set as Parser property